Databricks/ganglia/gconf/conf.d/gpu.pyconf

```diff
@@ -0,0 +1,142 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
```

Comment:
where were these files copied from?

Reply:
I actually wrote this file, and did not copy from anywhere.